src/main/java/bio/terra/service/dataset/AssetDao.java

```diff
@@ -131,7 +125,7 @@ public List<AssetSpecification> retrieveAssetSpecifications(Dataset dataset) {
           UUID specId = rs.getObject("id", UUID.class);
           AssetSpecification spec = new AssetSpecification().id(specId).name(rs.getString("name"));
           spec.assetTables(
-              new ArrayList(
+              new ArrayList<>(
```

Comment:
Could this use `List.of()` instead?
```suggestion
             List.of(
```

Reply:
I ended up pushing this conversion to list down to the private helper method, which follows other convention in the class.